compiler/src/dotty/tools/dotc/core/tasty/TastyFormat.scala

```diff
@@ -225,7 +227,7 @@ object TastyFormat {
 
   final val header = Array(0x5C, 0xA1, 0xAB, 0x1F)
   val MajorVersion = 2
-  val MinorVersion = 0
+  val MinorVersion = 1
```

Comment:
We do not need this since we have not published `2.0` yet

Reply:
No sure what you mean by "published". We do have nightly builds out that use 2.0 and we should bump the version when we change something because it's good practice and because it's valuable information to anyone wanting to make a separate implementation of tasty.

Reply:
OK